alvr/client_core/src/c_api.rs

```diff
@@ -41,11 +41,10 @@ pub struct AlvrClientCapabilities {
     encoder_high_profile: bool,
     encoder_10_bits: bool,
     encoder_av1: bool,
-    use_10bit: bool,
-    use_full_range: bool,
-    encoding_gamma: f32,
-    enable_hdr: bool,
-    enable_color_correction: bool,
+    preferred_use_10bit: bool,
+    preferred_use_full_range: bool,
+    preferred_encoding_gamma: f32,
+    preferred_enable_hdr: bool,
```

Comment:
Uhm, maybe I should have been more explicit, but you should make these names sound nicer: `prefer_10bit, prefer_full_range, prefer_hdr`. Ok for `preferred_encoding_gamma`.